packages/components/package.json

```
@@ -25,8 +25,10 @@
     "clean": "rimraf dist",
     "cleanAll": "rimraf dist && rimraf node_modules",
     "prepublishOnly": "yarn install && cross-env WEBPACK_STATS=errors-only yarn build",
-    "test": "cross-env NODE_ENV=test jest --maxWorkers=6 --silent",
-    "test-ci": "cross-env NODE_ENV=test jest --ci --silent",
+    "test": "npm run test-react ; npm run test-enzyme",
+    "test-enzyme": "cross-env NODE_ENV=test jest --config jest.enzyme.config.js --maxWorkers=6 --silent",
+    "test-react": "cross-env NODE_ENV=test jest --config jest.react.config.js",
+    "test-ci": "cross-env NODE_ENV=test jest --config jest.enzyme.config.js --ci --silent",
```

Maybe make `test-ci` just run them both so TeamCity doesn't have to worry about the different test types?
```suggestion
    "test-ci": "npm run test-react ; cross-env NODE_ENV=test jest --config jest.enzyme.config.js --ci --silent",
```

---

I didn't make exactly the suggested change but this appears to work well in reporting in CI!